generate-stackbrew-library.sh

```diff
@@ -13,7 +13,7 @@ declare -A debianSuite=(
 defaultAlpineVersion='3.7'
 declare -A alpineVersion=(
 	[1.8]='3.5'
-	[1.9]='3.6'
+	[1.9]='3.7'
```

Comment:
This one should be left at 3.6 -- this controls what `golang:1.9-alpine` points to, which needs to stay at 3.6.